### DISCUSSION TITLE
Ability to print backup passphrase as QR code

### DISCUSSION THREAD

**Discussion by ShadowJonathan** [element-hq/element-meta#702] [Ideas]
#### What would you like to do?

Be able to print backup keys/passphrase as a (highly error resistant) QR code.

#### Why would you like to do it?

Users might want to be able to store/backup their backup key as something easily scannable, such as a QR code. The advantage over having them write down numbers is that QR codes can become much more resistant to tears/wear with high [error correction](https://www.qrcode.com/en/about/error_correction.html).

#### How would you like to achieve it?

Be able to print the keys upon backup configuration, and be able to scan them via the same method.

### Additional context

This came up in the Matrix stand at FOSDEM 2022 :)

**Comment by HarHarLinks** [comment]
> much more resistant to tears/wear with high [error correction](https://www.qrcode.com/en/about/error_correction.html).

downside of this is that half of the code might be enough to read the whole code, so it needs special design. I.e. you can't rip it up and throw it away, but need to burn it.

**Comment by t3chguy** [comment]
The QR format would need speccing so that other clients can create/consume it too.

**Comment by kittykat** [comment]
Moving to discussions as this is cross-platform :+1:
